lib/ReactViews/Map/Legend/LocationBar.jsx

```diff
@@ -23,7 +24,9 @@ const LocationBar = observer(
     },
 
     toggleUseProjection() {
-      this.props.mouseCoords.toggleUseProjection();
+      runInAction(() => {
```

Comment:
Just a small nitpick - It might be better to wrap `MouseCoords.toggleUseProjection` with an `@action` decorator instead.